commands/info.md

```diff
@@ -291,10 +291,11 @@ Here is the meaning of all fields in the **stats** section:
 *   `total_error_replies`: Total number of issued error replies, that is the sum of
     rejected commands (errors prior command execution) and
     failed commands (errors within the command execution)
-*    `total_reads_processed`: Total number of read events processed
-*    `total_writes_processed`: Total number of write events processed
-*    `io_threaded_reads_processed`: Number of read events processed by the main and I/O threads
-*    `io_threaded_writes_processed`: Number of write events processed by the main and I/O threads
+*   `dump_payload_sanitizations`: Total number of deep dump payloads integrity validations.
```

Comment:
maybe we should add some reference to the `sanitize-dump-payload` config so users have a chance to understand what this is about.

Reply:
yes, good point

Adding this: 
> See `dump_payload_sanitizations` configuration in config file.

or 
> See `dump_payload_sanitizations` configuration in config file for more details.

somethings like this

Reply:
```suggestion
*   `dump_payload_sanitizations`: Total number of deep dump payloads integrity validations (see `sanitize-dump-payload` config).
```

Reply:
the one with the `_` is the variable.. we're referring to the user config name.